static/js/components/EducationForm.js

```diff
@@ -51,36 +47,86 @@ class EducationForm extends ProfileFormFields {
+    // {valueSelected: String(inclusions[level.value])} :
+    // {};
+    let radioIconStyle = {'marginRight': '8px'};
+    return (
+      <RadioButtonGroup
+        className={`profile-radio-switch ${level.value}`}
+        id={`profile-tab-education-switch-${level.value}`}
+        name={`profile-tab-education-switch-${level.value}`}
+        onChange={(event, value)=> this.handleRadioClick(value, level.value)}
+        {...valueSelectedPropValue}
+      >
+        <RadioButton value={"true"} label="Yes" iconStyle={radioIconStyle} style={{'marginRight': '30px'}} />
+        <RadioButton value={"false"} label="No" iconStyle={radioIconStyle} style={{'marginRight': '15px'}} />
+      </RadioButtonGroup>
+    );
+  };
+
+  handleRadioClick(value: string, level: string): void {
+    if (value === "true") {
+      this.openNewEducationForm(level, null);
+    }
+  }
+
+  renderEducationQuestionForm(level: Option): React$Element<*> {
+    let label = level.label,
+      prefix = label.toLowerCase().startsWith("a") ? "an" : "a",
+      levelName = !label.endsWith("degree") ? `${label.toLowerCase()} degree` : label.toLowerCase();
+    return <Cell col={12} className="profile-card-header profile-form-row">
+        <span>
+          {`Do you have ${prefix} ${levelName}?`}
+        </span>
+        { this.educationLevelRadioSwitch(level) }
+      </Cell>;
+  }
+
+  renderEducationLevelEntries(level: Option): Array<React$Element<*>|void>|void {
     const {
-      ui: { educationDegreeInclusions },
-      profile: { education },
+      profile: { education }
     } = this.props;
-    if (educationDegreeInclusions[level.value]) {
-      let rows: Array<React$Element<*>|void> = [];
-      if (education !== undefined) {
-        let sorted = educationEntriesByDate(education);
-        rows = sorted.filter(([,entry]) => (
-          entry.degree_name === level.value
-        )).map(([index, entry]) => this.educationRow(entry, index));
-      }
-      rows.push(
-        <Cell col={12} className="profile-form-row" key={"I'm unique!"}>
-          <a
-            className="mm-minor-action"
-            onClick={() => this.openNewEducationForm(level.value, null)}
-          >
-            Add another
-          </a>
-        </Cell>
-      );
-      return rows;
+    let rows: Array<React$Element<*>|void> = [];
+    if (education !== undefined) {
+      let sorted = educationEntriesByDate(education);
+      rows = sorted.filter(([,entry]) => (
+        entry.degree_name === level.value
+      )).map(([index, entry]) => this.educationRow(entry, index));
+    }
+    rows.unshift(
+      <Cell col={12} className="profile-form-row" key={`header-row`}>
+        <strong>{level.label}</strong>
+      </Cell>
+    );
+    rows.push(
+      <Cell col={12} className="profile-form-row" key={`add-row`}>
+        <a
+          className="mm-minor-action"
+          onClick={() => this.openNewEducationForm(level.value, null)}
+        >
+          Add another
```

Comment:
While we're in here, do we want to change this to "Add degree" 

@roberthouse54 @Ferdi ?